services/permits/app/extract_and_validate_pdf.py

```diff
@@ -43,34 +44,69 @@ def authenticate_with_oauth():
     return oauth_session
 
 
+def refresh_token(oauth_session):
```

Comment:
Added automatic refreshing of auth token here as it would sometimes timeout due to how long the extraction process takes